shared/constants/types/wallets.tsx

```diff
@@ -275,6 +275,27 @@ export type _AirdropDetails = {
 }
 export type AirdropDetails = I.RecordOf<_AirdropDetails>
 
+export type TrustlineAssetID = string
+export const makeTrustlineAssetID = (issuerAccountID: string, assetCode: string): TrustlineAssetID =>
+  `${issuerAccountID}-${assetCode}`
+
+export type _TrustlineAsset = {
+  assetCode: string
+  issuerAccountID: string
+  issuerVerifiedDomain: string
+  trustedLimit: number // 0 if not trusted
+}
+export type TrustlineAsset = I.RecordOf<_TrustlineAsset>
+
+export type _Trustline = {
+  acceptedAssets: I.List<TrustlineAssetID>
+  assetMap: I.Map<TrustlineAssetID, TrustlineAsset>
+  expandedAssets: I.Set<TrustlineAssetID>
+  popularAssets: I.List<TrustlineAssetID>
+  searchingAssetsHit: I.List<TrustlineAssetID>
+}
+export type Trustline = I.RecordOf<_Trustline>
```

Comment:
I'm changing these structures in my next PR, so might as well skip these for now and continue to the actual components.